extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -639,6 +639,27 @@ public boolean shouldUseIterable() {
         return serviceVersions;
     }
 
+    private Map<String, String> llcProperties;
+
+    public Map<String, String> getLlcProperties() {
+        if (llcProperties == null) {
+            llcProperties = new HashMap<>();
+        }
+        llcProperties.put("artifact-id", getArtifactId());
+        llcProperties.put("service-name", getServiceName());
```

Comment:
how is this different from setting `--title=<custom client name>`?

Reply:
It is used in resource files like `pom`, `readme`.